src/appShell/App/Container.tsx

```diff
@@ -36,10 +36,7 @@ interface IContainerProps {
 
 function isLocalDBServer() {
     try {
-        return (
-            (window as any).frontendConfig.serverConfig.app_name ===
-            'localdbe2e'
-        );
+        return (window as any).frontendConfig.serverConfig.is_localdb_server;
```

Comment:
in this particular case i don't think using app_name is too bad (kind of unlikely that people will use the same name). This parameter is maybe cleaner but in that case we also want to set `is_localdb_server` when we are running the localdb tests. It seems like more work than it's worth so maybe ok to leave as is